vuepress/docs/next/tutorials/create/ms/add-access-controls.md

```diff
@@ -1,109 +1,167 @@
+3. Verify that attempting to delete via the UI generates a `403 error` in the browser console and an error in the service logs similar to the following:
 ```
 2021-03-22 15:56:16.205  WARN 3208 --- [  XNIO-1 task-3] o.z.problem.spring.common.AdviceTraits   : Forbidden: Access is denied
 ```
-That's exactly what we wanted! This demonstrates that a user without `conference-admin` is unable to call the delete API.
 
-Next, let's update the MFE so a user without the `conference-admin` authority cannot even see the delete button in the UI.
+This demonstrates that a user without `conference-admin` is unable to call the delete API.
+
+### Step 8: Hide the delete button
 
-14. Edit the `ConferenceTableContainer.js` under `ui/widgets/conference/tableWidget/src/components`. Replace the onDelete logic with an additional check on the user's authorities.
+The MFE UI can be updated to hide the delete button from a user without the `conference-admin` authority. The key logic checks whether the `internal` client role `conference-admin` is mapped to the current user via the hasResourceRole call.
+
+1. Go to the `ui/widgets/conference/tableWidget/src/components` directory
+2. Open `ConferenceTableContainer.js` 
+3. Replace the `onDelete` logic with an additional user permission:
 ```
     const isAdmin = (keycloak && keycloak.authenticate) ? keycloak.hasResourceRole("conference-admin", "internal"): false;
     const showDelete = onDelete && isAdmin;
 
     const Actions = ({ item }) =>
       showDelete ? (
 ```
+4. Confirm that the delete icon is no longer visible in the MFE, which should have automatically reloaded
 
-The key logic there is the hasResourceRole call which checks whether the `internal` client role `conference-admin` was mapped to the current user.
-
-15. View the MFE (whch should have automatically reloaded) and you should see that the delete icon is no longer visible, matching the admin's current permissions.  We've now verified that a user with just `conference-user` can neither see the delete action in the UI nor call its corresponding API.
+### Step 9: Grant and verify delete permissions
 
-Next, let's promote the admin user to a full `conference-admin` so they can delete Conferences.
+Promote the admin user to a full `conference-admin` to reinstate the ability to delete Conferences.
 
-16. Go back into Keycloak at <http://localhost:9080>, then go to `Users → View all users → admin → Role Mappings`, and also give the user the `conference-admin` role.
-
-17. Reload the MFE. The delete icons should now be visible and you should be able to successfully delete a Conference from the list. This satisfies our original business requirement. 
+1. Return to Keycloak at <http://localhost:9080>
+2. Go to `Users` → `View all users` → `admin` → `Role Mappings`
+3. Give the user the `conference-admin` role
+4. Reload the MFE 
+5. Confirm the delete icon is visible and you should be able to 
```

Comment:
Sentence fragment here. Just drop "and you should be able to"

Reply:
*facepalm*